applications/gafaelfawr/values-usdfdev.yaml

```diff
@@ -0,0 +1,226 @@
+      - rubin_users-g
+      - rubin_users-h
+      - rubin_users-i
+      - rubin_users-j
+      - rubin_users-k
+      - rubin_users-l
+      - rubin_users-m
+      - rubin_users-n
+      - rubin_users-o
+      - rubin_users-p
+      - rubin_users-q
+      - rubin_users-r
+      - rubin_users-s
+      - rubin_users-t
+      - rubin_users-u
+      - rubin_users-v
+      - rubin_users-w
+      - rubin_users-x
+      - rubin_users-y
+      - rubin_users-z
+      - rubin_admin_datasets
+      - rubin_admin_repos
+      - "unix-admin"
+    "exec:user":
+      - "lsst"
+      - lsst-ccs
+      - rubin_users
+      - rubin_users-a
+      - rubin_users-b
+      - rubin_users-c
+      - rubin_users-d
+      - rubin_users-e
+      - rubin_users-f
+      - rubin_users-g
+      - rubin_users-h
+      - rubin_users-i
+      - rubin_users-j
+      - rubin_users-k
+      - rubin_users-l
+      - rubin_users-m
+      - rubin_users-n
+      - rubin_users-o
+      - rubin_users-p
+      - rubin_users-q
+      - rubin_users-r
+      - rubin_users-s
+      - rubin_users-t
+      - rubin_users-u
+      - rubin_users-v
+      - rubin_users-w
+      - rubin_users-x
+      - rubin_users-y
+      - rubin_users-z
+      - rubin_admin_datasets
+      - rubin_admin_repos
+      - "unix-admin"
+    "read:tap":
+      - "lsst"
+      - lsst-ccs
+      - rubin_users
+      - rubin_users-a
+      - rubin_users-b
+      - rubin_users-c
+      - rubin_users-d
+      - rubin_users-e
+      - rubin_users-f
+      - rubin_users-g
+      - rubin_users-h
+      - rubin_users-i
+      - rubin_users-j
+      - rubin_users-k
+      - rubin_users-l
+      - rubin_users-m
+      - rubin_users-n
+      - rubin_users-o
+      - rubin_users-p
+      - rubin_users-q
+      - rubin_users-r
+      - rubin_users-s
+      - rubin_users-t
+      - rubin_users-u
+      - rubin_users-v
+      - rubin_users-w
+      - rubin_users-x
+      - rubin_users-y
+      - rubin_users-z
+      - rubin_admin_datasets
+      - rubin_admin_repos
+      - "unix-admin"
+    "read:image":
+      - "lsst"
+      - lsst-ccs
+      - rubin_users
+      - rubin_users-a
+      - rubin_users-b
+      - rubin_users-c
+      - rubin_users-d
+      - rubin_users-e
+      - rubin_users-f
+      - rubin_users-g
+      - rubin_users-h
+      - rubin_users-i
+      - rubin_users-j
+      - rubin_users-k
+      - rubin_users-l
+      - rubin_users-m
+      - rubin_users-n
+      - rubin_users-o
+      - rubin_users-p
+      - rubin_users-q
+      - rubin_users-r
+      - rubin_users-s
+      - rubin_users-t
+      - rubin_users-u
+      - rubin_users-v
+      - rubin_users-w
+      - rubin_users-x
+      - rubin_users-y
+      - rubin_users-z
+      - rubin_admin_datasets
+      - rubin_admin_repos
+      - "unix-admin"
+
+  initialAdmins:
+    - "afausti"
+    - "athor"
+    - "cbanek"
+    - "frossie"
+    - "jonathansick"
+    - "rra"
+    - "simonkrughoff"
+    - "ytl"
+    - "ppascual"
+
+cloudsql:
+  enabled: false
+
```

Comment:
ErrorFooter commenting to report problems to rubin-dp0 was removed here. Any discussion on where to direct issues?

Reply:
That's not something we in SQuaRE would know -- that's up to you folks who are running the USDF. A lot of the support seems to be in #usdf-ops on Slack, but I don't know if that's what you want to say here.